src/util/desktop-notifier.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+import defaultNotifier from 'node-notifier';
+
+import {createLogger} from './logger';
+
+const log = createLogger(__filename);
+
+type desktopNotificationsParams = {
+  title: string,
+  message: string,
+  icon?: string,
+};
+
+type desktopNotificationsOpts = {
+  notifier?: typeof defaultNotifier,
+  logger?: typeof log,
+};
+
+export function desktopNotifications(
+  {
+    title, message, icon,
+  }: desktopNotificationsParams,
+  {
+    notifier = defaultNotifier,
+    logger = log,
+  }: desktopNotificationsOpts = {}
+): void {
+
+  notifier.notify({title, message, icon}, (err, res) => {
```

Comment:
Logging the error looks good, thanks, but this introduces a synchronization problem for testing. You will need to return a promise instead (just for testing purposes), like:
````javascript
return new Promise((resolve, reject) => {
  notifier.notify({title, message, icon}, (err, res) => {
    if (err) {
      logger.debug(`notifier error: ${err.message},` +
                   ` response: ${res}`);
      reject(err);
    } else {
      resolve();
    }
  });
});
````